solutions/usaco-966.mdx

```diff
@@ -47,4 +47,39 @@ ans += first8[N % 8]
 print(ans)
 ```
 </PySection>
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+
+int main() {
+	// your code goes here
+	int n;
+	cin >> n;
+	
+	// vector the first 8. 
+    //14 is in the zeroth position     since if were trying to find the 
```

Comment:
```suggestion
     // vector the first 8. 
    //14 is in the zeroth position    
    // Since if were trying to find the eighth number, it will be 8%8=0
```